pkg/kubelet/kubelet_node_status.go

```diff
@@ -52,6 +52,9 @@ func (kl *Kubelet) registerWithAPIServer() {
 	if kl.registrationCompleted {
 		return
 	}
+
+	kl.nodeStartupLatencyTracker.RecordAttemptRegisterNode()
```

Comment:
is it possible that if this succeeds this never gets called?

https://github.com/kubernetes/kubernetes/blob/91344b4008f6c5dfce82541b0c96cb8670b8350c/pkg/kubelet/kubelet_node_status.go#L494